runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -146,6 +148,12 @@ public boolean isEmpty() {
         return mainMap.isEmpty();
     }
 
+    @Accessor
+    public boolean hasIndices() {return !indexFunctions.isEmpty();}
+
+    @Getter
+    public Class indexerClass;
```

Comment:
it doesn't "compile". CorfuMap#CORFUSMR.java is complaining about not knowing the symbol class F